README.md

```diff
@@ -1 +1,36 @@
-# capstone-1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Microverse Portfolio
+
+> a website about the DeveloperWeek event. for the first microverse capstone.
+
+## Built With
+
+- HTML
+- SCSS
+- JavaScript
+
+## Live Link
+
+[Live Link](https://rachidelaid.github.io/capstone-1/)
+
```

Comment:
- [ ] Please fix this link, it does not work 

![image](https://user-images.githubusercontent.com/77038610/147334284-df4d18e8-5457-40ee-a542-8dbb63aabf28.png)